sdcm/cluster.py

```diff
@@ -110,6 +110,8 @@ class Setup:
     INTRA_NODE_COMM_PUBLIC = False
     RSYSLOG_ADDRESS = None
     DECODING_QUEUE = None
+    USE_LEGACY_CLUSTER_INIT = False
```

Comment:
Please document this parameter somewhere. Can be a comment here. It should explain what the "legacy method" of cluster initialization is.

Reply:
it is documented in sct_config...

Reply:
Also it could find here docs/configuration_options.md

Reply:
Ok, thanks